sisyphus/tools.py

```diff
@@ -208,7 +209,8 @@ def sh(command,
 
     try:
         if capture_output:
-            return subprocess.check_output(command, shell=True, executable=executable).decode()
+            return subprocess.check_output(command, shell=True, executable=executable,
+                                           stderr=subprocess.STDOUT if include_stderr else subprocess.DEVNULL).decode()
```

Comment:
The current default behaviour is that not captured output still written to the logfile. Setting stderr=subprocess.DEVNULL will suppress it. Let's stay with stderr=None if it's include_stderr is false.
LGTM beside that.